src/telliot_feeds/sources/mimicry/mashup_source.py

```diff
@@ -171,6 +175,9 @@ async def fetch_tokens_mcap(self) -> Optional[Any]:
             if response is not None:
                 try:
                     market_cap = response["market_data"]["market_cap"][self.currency.lower()]
+                    if not market_cap:
+                        logger.debug(f"No value fetched for token: {response['name']}")
```

Comment:
In this case, you're sure `response` will always contain the `name` key?

Reply:
yeah you're right about this.